bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```diff
@@ -192,6 +192,7 @@ def softlayer_stages
           :bosh_harden,
           :bosh_enable_password_authentication,
           :bosh_softlayer_agent_settings,
+          :bosh_config_root_ssh_login,
```

Comment:
@mattcui i would love to have a discussion about how to remove password auth and replace it with ssh keys for softlayer stemcell. hopefully @maximilien can help us arrange a call?

Reply:
Can we plan this tomorrow Thursday September 28th, 2017 @ 5:30p PDT?

Beijing team comment here. We might also be able to do 6p. If yes, I'll schedule Zoom.

Best,

max

Reply:
@maximilien sorry that I have already on vacation and holiday since today, I will not be back until Oct. 9, so have to talk after the vacation.